src/app/cli/src/graphql.ml

```diff
@@ -431,6 +435,10 @@ module Make (Commands : Coda_commands.Intf) = struct
             ; fee ~doc:"Fee amount in order to send a stake delegation"
             ; memo ~doc:"Public description of a stake delegation" ]
 
+      let delete_wallet =
+        obj "deleteWallet" ~coerce:Fn.id
+          ~fields:[arg "publicKey" ~typ:(non_null string)]
```

Comment:
It looks like you defined this twice?

Reply:
oh no, one of them is defined in inputs and one of them is defined in payload